plugins/score-card/src/api/types.ts

```diff
@@ -55,10 +65,58 @@ export enum ScoreSuccessEnum {
 }
 
 // TODO: decide what with this interface. It makes the API to be tight coupled with catalog API (the component is coupled anyway). Shall this be internal implementation?
+/**
+ * @deprecated Use EntityScoreArea instead
+ */
 export interface SystemScoreExtended extends SystemScore {
   catalogEntity: Entity | undefined;
   catalogEntityName: CompoundEntityRef | undefined;
   owner: CompoundEntityRef | undefined;
   reviewer: CompoundEntityRef | undefined;
   reviewDate: Date | undefined;
 }
+
+
+export interface EntityScore {
+  /**
+   * @deprecated Use entityName instead
```

Comment:
Why deprecated if we still support this in the config as `useEntityRef: false` option?

Reply:
I still  needed `systemEntityName` on the new type to make it backward compatible, but maybe there was another way in TS I couldn't find.

It won't be relevant anymore after dropping the backward compatibility.